packages/jest-cli/src/cli/index.js

```diff
@@ -67,6 +68,12 @@ const runCLI = async (
       outputStream,
     );
 
+    if (argv.clearCache) {
+      clearDirectory(configs[0].cacheDirectory);
+      process.stdout.write(`Cleared ${configs[0].cacheDirectory}\n`);
+      process.exit(0);
+    }
```

Comment:
We should be able to clear the cache for certain project or all projects, right now it's only for the first one.

Reply:
I wasn't sure how this worked: should I just map over the configs array?

Reply:
yup, should be fine